helm-chart/renku/requirements.yaml

```diff
@@ -36,6 +36,9 @@ dependencies:
   version: 16.0.4
   repository: "https://codecentric.github.io/helm-charts"
   condition: keycloak.enabled
+- name: certificates
+  version: 0.11.0-rc2
```

Comment:
Should we version the charts independently and tag `0.1.0`?

Reply:
@ableuler I am not really sure how we can version the certificates library chart independently if it is part of the same chartpress file as the renku chart. Also even if we used separate chartpress files for the renku chart and for the certificates library chart then we would have to setup github actions that trigger on a specific tag formatted in such a way so that it does not trigger a renku release but it triggers a certificates helm chart release? I was trying to think about all of this but then decided that versioning the certificates helm chart and renku together is the easiest thing to do.

Let me know what you think. And if you do think that they should not be released together then how would you setup the CI for this?

Reply:
The certificates library chart does already have its own chartpress file in the certificates folder, no? I think the easiest would be to not bother about building and publishing the library chart automatically and just run `chartpress --version `X.Y.Z` --publish` manually from the certificates folder whenever we change that chart, and then update the chart dependency in the renku chart requirements.yaml accordingly.

Reply:
So now the images are built and published manually. Chartpress would not work because it expects the image versions to be updated in the helm chart but the library helm chart has not values file and it crashes chartpress.

If I put the images in the renku helm chart chartpress file then new images for the certificates are built and updated in the values file on every renku release. And I think this is confusing and unnecessary. This kinda pushes us toward the certificates releases being tied to renku - which I really dont want. Ideally (and I think this will be the case) the images for the certificates rarely change.